apps/marketplace/components/Brief/BriefDownloadResponses.js

```diff
@@ -1,26 +1,33 @@
 import React, { Component } from 'react'
+import { Redirect } from 'react-router-dom'
+import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 import AUheading from '@gov.au/headings/lib/js/react.js'
-import AUbutton from '@gov.au/buttons/lib/js/react.js'
 import AUpageAlert from '@gov.au/page-alerts/lib/js/react.js'
+import format from 'date-fns/format'
 import { rootPath } from 'marketplace/routes'
-import { getResponsesFileSizeAndType } from 'marketplace/components/helpers'
+import { getResponsesFileSizeAndType, hasPermission } from 'marketplace/components/helpers'
 import styles from './BriefDownloadResponses.scss'
 
 export class BriefDownloadResponses extends Component {
   constructor(props) {
     super(props)
-
     this.handleButtonClick = this.handleButtonClick.bind(this)
   }
 
-  handleButtonClick(e) {
-    e.preventDefault()
-    window.location = `/api/2/brief/${this.props.brief.id}/respond/documents`
+  handleButtonClick() {
+    setTimeout(() => {
+      this.props.reloadBrief()
```

Comment:
It's just cosmetic, but this is probably more accurate if called `this.props.onDownloadClick()`.

Reply:
updated